setup.cfg

```diff
@@ -38,25 +38,30 @@ package_dir = =src
 packages = find:
 
 [options.extras_require]
+; This is list of minimum dependencies required to run the test.
+; The plan is to remove the `test` extra requirement and skip tests that
+; depend on then
 test =
     cython-test-exception-raiser ~= 1.0
     PyHamcrest >= 1.9.0
+
+; List of dependencies required to build the documentation and test the
+; release scripts and process.
+dev_release =
+    towncrier >= 17.4.0
+    pydoctor >= 20.7.0
+    sphinx-rtd-theme~=0.5.0
+    readthedocs-sphinx-ext~=2.1
+    sphinx~=3.3
+
+; All the extra tools used to help with the development process.
 dev =
+    %(dev_release)s
     pyflakes >= 1.0.0
     twisted-dev-tools >= 0.0.2
     python-subunit
-    towncrier >= 17.4.0
     twistedchecker >= 0.7.2
-    # force upgrades for rtd default packages: https://git.io/JU73V
-    alabaster~=0.7.12
-    commonmark~=0.9.1
-    docutils~=0.16.0
-    mock~=4.0
-    pillow~=7.2
-    readthedocs-sphinx-ext~=2.1
-    recommonmark~=0.6.0
-    sphinx~=3.2
-    sphinx-rtd-theme~=0.5.0
```

Comment:
so we're going to be using the old deps now? https://github.com/readthedocs/readthedocs.org/blob/046c4bd20fd875807cb3e1840846249160c37079/readthedocs/doc_builder/python_environments.py#L354-L389

Reply:
```
readthedocs=
    %(dev_release)s
    alabaster~=0.7.12
    commonmark~=0.9.1
    docutils~=0.16.0
    mock~=4.0
    pillow~=7.2
    recommonmark~=0.6.0
    sphinx~=3.3
```

and then update the `extra` in `./readthedocs.yml`